tests/integration/kubernetes/gha-run.sh

```diff
@@ -532,8 +532,19 @@ function deploy_nydus_snapshotter() {
 
 	echo "::endgroup::"
 	echo "::group::nydus snapshotter logs"
-	pods_name=$(kubectl_retry get pods --selector=app=nydus-snapshotter -n nydus-system -o=jsonpath='{.items[*].metadata.name}')
-	kubectl_retry logs "${pods_name}" -n nydus-system
+	get_pod_name="kubectl_retry get pods --selector=app=nydus-snapshotter -n nydus-system -o=jsonpath='{.items[*].metadata.name}'"
+	pods_name=$(eval "${get_pod_name}")
+	while ! kubectl_retry logs "${pods_name}" -n nydus-system; do
```

Comment:
Hi @BbolroC !

Is there a reason to not use `kubectl_retry logs --selector=app=nydus-snapshotter -n nydus-system ` instead of getting the pod name between retrievals?